themes/kelpie/_site-header.scss

```diff
@@ -1,8 +0,0 @@
-/*--------------------------------------------------------------
-# Site Footer
```

Comment:
I don't think this file was in use. If it was, it should be moved! 😆

Reply:
Yes yes yes thank you. I keep accidentally opening it, wondering why it exists, and then losing track and not being able to find where it is. 🙄